inc/html.php

```diff
@@ -954,38 +362,11 @@ function html_list_index($item){
  *
  * @param array $item
  * @return string html
+ * @deprecated 2020-07-18
  */
-function html_li_index($item){
-    global $INFO;
-    global $ACT;
-
-    $class = '';
-    $id = '';
-
-    if($item['type'] == "f"){
-        // scroll to the current item
-        if(isset($INFO) && $item['id'] == $INFO['id'] && $ACT == 'index') {
-            $id = ' id="scroll__here"';
-            $class = ' bounce';
-        }
-        return '<li class="level'.$item['level'].$class.'" '.$id.'>';
-    }elseif($item['open']){
-        return '<li class="open">';
-    }else{
-        return '<li class="closed">';
-    }
-}
-
-/**
- * Default List item
- *
- * @author Andreas Gohr <andi@splitbrain.org>
- *
- * @param array $item
- * @return string html
- */
-function html_li_default($item){
-    return '<li class="level'.$item['level'].'">';
+function html_li_index($item) {
+    dbg_deprecated(\dokuwiki\Ui\Index::class .'::tagListItem()');
+    return (new dokuwiki\Ui\Index)->tagListItem($item);
 }
 
```

Comment:
```suggestion
/**
 * Default List item
 *
 * @author Andreas Gohr <andi@splitbrain.org>
 *
 * @param array $item
 * @return string html
 * @deprecated 2020-07-18
 */
function html_li_default($item){
    return '<li class="level'.$item['level'].'">';
}
```
Gives still another release mercy to the users of this default callback.

Reply:
Ok, the **html_li_default()** has declared again to call explicitly `dbg_deprecated()`. see 89b939d